src/sandstorm/spk.c++

```diff
@@ -63,6 +63,14 @@ typedef kj::byte byte;
 static const uint64_t APP_SIZE_LIMIT = 1ull << 30;
 // For now, we will refuse to unpack an app over 1 GB (decompressed size).
 
+static const uint32_t MAX_DEFINED_APIVERSION = 0;
+// The maximum API version that has been defined, as of this source code's compilation.
+// We should outright refuse to pack an app claiming a newer minApiVersion than this, because we
+// can't possibly know what the constraints are on that API.
+//
+// We should additionally warn if the user attempts to pack an app claiming a newer maxApiVersion
```

Comment:
Fixed.  Sorry about the force-push; I noticed the bad comment myself and thought I'd sneak in a correction in before anyone noticed, but I was wrong.